app/src/main/java/com/hammer/talkbbokki/domain/di/UseCaseModule.kt

```diff
@@ -0,0 +1,20 @@
+package com.hammer.talkbbokki.domain.di
+
+import com.hammer.talkbbokki.domain.repository.TopicRepository
```

Comment:
🚫 **[ktlint]** <import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between